thirdparty/README.md

```diff
@@ -393,6 +393,23 @@ Files extracted from the upstream source:
 - All .h files in `src/`
 - LICENSE.txt
 
+##open-simplex-noise
+
+- Upstream: https://github.com/smcameron/open-simplex-noise-in-c
+- Version: git (0d555e7, 2018)
+- License: The Unlicense
```

Comment:
The short identifier is just "Unlicense": https://spdx.org/licenses/Unlicense.html